common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -2702,17 +2703,28 @@ object CelebornConf extends Logging {
       .intConf
       .createWithDefault(512)
 
-  val CLIENT_PUSH_MAX_REQS_IN_FLIGHT: ConfigEntry[Int] =
-    buildConf("celeborn.client.push.maxReqsInFlight")
+  val CLIENT_PUSH_MAX_REQS_IN_FLIGHT_TOTAL: ConfigEntry[Int] =
+    buildConf("celeborn.client.push.maxReqsInFlight.total")
       .withAlternative("celeborn.push.maxReqsInFlight")
```

Comment:
From my understanding, it should be the alternative of `celeborn.client.push.maxReqsInFlight.perWorker` rather than `...total`?

Reply:
Not exactly. ```eleborn.client.push.maxReqsInFlight.total``` is equivalent to ```celeborn.push.maxReqsInFlight``` in 0.2, which means global limit of in flight requests. ```celeborn.client.push.maxReqsInFlight.perWorker``` is newly introduced in 0.3

Reply:
I know, the conclusions are based on different the baseline assumption. I assume this PR targets to 0.3.1 and the baseline is 0.3.0

Reply:
Resolve this conversation since we are going to fail 0.3.0-rc1